README.md

```diff
@@ -4,247 +4,148 @@ This backend is currently in development and is not advised for Production workf
-            "retryWrites": "true",
-            "w": "majority",
-            "tls": "false",
-        },
-    },
-}
-```
+- **Database Connectivity**  
+    
+  - Directly tune MongoDB connection settings within your Django configuration\!  
+  - Work against a persisted cloud instance of MongoDB for free\!
 
-For a localhost configuration, you can omit `HOST` or specify
-`"HOST": "localhost"`.
 
-`HOST` only needs a scheme prefix for SRV connections (`mongodb+srv://`). A
-`mongodb://` prefix is never required.
+- **Model MongoDB Documents Through Django’s ORM**  
+    
+  - Translate Django model instances to MongoDB documents.  
+  - Create new collections corresponding to models.  
+  - Supports field validation, data storage, updating, and deletion.  
+  - Maps Django's built-in fields to MongoDB data types.  
+  - Provides new custom fields for arrays (ArrayField) and embedded documents (EmbeddedModelField).  
+  - Supports core migration functionalities including creating, deleting, and updating indexes and collections
 
-`OPTIONS` is an optional dictionary of parameters that will be passed to
-[`MongoClient`](https://pymongo.readthedocs.io/en/stable/api/pymongo/mongo_client.html).
 
-`USER`, `PASSWORD`, and `PORT` (if 27017) may also be optional.
+- **Index Management**  
+    
+  - Create single, compound, and unique indexes using Django Indexes  
+  - Create MongoDB partial indexes in Django using Q notation.
 
-For a replica set or sharded cluster where you have multiple hosts, include
-all of them in `HOST`, e.g.
-`"mongodb://mongos0.example.com:27017,mongos1.example.com:27017"`.
 
-Alternatively, if you prefer to simply paste in a MongoDB URI rather than parse
-it into the format above, you can use:
+- **Querying Data**  
+    
+  - Querying API powered through the amazing MongoDB Aggregation Pipeline  
+  - Supports most functions of the Django QuerySet API  
+  - Support Query Annotations and common SQL AGGREGATE operators  
+  - We support foreign keys and execute JOIN operations all in 1 database call  
+  - Through our custom raw\_aggregate call, MQL operations like Vector Search, Atlas Search, and GeoSpatial querying still yield Django QuerySet results\!
 
-```python
-import django_mongodb_backend
 
-MONGODB_URI = "mongodb+srv://my_user:my_password@cluster0.example.mongodb.net/myDatabase?retryWrites=true&w=majority&tls=false"
-DATABASES["default"] = django_mongodb_backend.parse_uri(MONGODB_URI)
-```
+- **Administrator Dashboard & Authentication**  
+    
+  - Manage your data in Django’s admin site.  
+  - Fully integrated with Django's authentication framework.  
+  - Supports native user management features like creating users and sessions.
 
-This constructs a `DATABASES` setting equivalent to the first example.
 
-#### `django_mongodb_backend.parse_uri(uri, conn_max_age=0, test=None)`
+- **Management Commands**  
+    
+  - Use commands like `migrate`, `makemigrations`, `flush`, `sqlmigrate`, many more.
```

Comment:
You already mentioned support for schema management. I don't think management commands need a special callout. Particularly not flush.